src/libespeak-ng/phoneme.h

```diff
@@ -249,7 +252,7 @@ phoneme_add_feature(PHONEME_TAB *phoneme,
 // Several phoneme tables may be loaded into memory. phoneme_tab points to
 // one for the current voice
 extern int n_phoneme_tab;
-extern PHONEME_TAB *phoneme_tab[N_PHONEME_TAB];
+extern const PHONEME_TAB *phoneme_tab[/*N_PHONEME_TAB*/];
```

Comment:
The "commented" code, I assume?  This is part of the change to make phoneme_list dynamic; I needed to ensure that code outside phonemelist.c was const, but it doesn't matter at present because I think PHONEME_TAB* is outside the phoneme_list control flow; it's only in synthdata.c (as the comment I added says).